src/containers/Tenant/Preview/Preview.scss

```diff
@@ -52,7 +52,16 @@
     &__result {
         overflow: auto;
 
+        // This fixes last row display for ordinary preview (not fullscreen)
         height: calc(100% - 40px);
         padding: 0 10px;
     }
 }
+
+// Fix white space footer block for fullscreen preview
+
+.kv-fullscreen {
+    .kv-preview__result {
+        height: 100%;
+    }
+}
```

Comment:
you can write it right next to the not-fullscreen property
`.kv-fullscreen & { ... }`
would be more isolated and easier to maintain

but it's ok either way
resolve this comment or update the pr, and I'll approve